### PR TITLE
feat(@desktop/metrics): Add metrics management page

### LINKED ISSUE #15490: Implement design for the Opt-in metric checkbox
### Description

 Follow up of https://github.com/status-im/status-desktop/issues/15446

Onboarding:
https://www.figma.com/design/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=40573-92941&t=1aCmIYrch9JqLCIa-0

Settings:
https://www.figma.com/design/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=24721-8826&t=J9tvSFrAc93owo5j-0

### Acceptance Criteria

- The backend is integrated with the UI
- The UI is implemented as per designs (needs a new settings page)

### PR DESCRIPTION
**Pull request by endulab**
Add new settings page for metrics management - Privacy and security
Add popup to enable/disable metrics on Onboarding and Privacy screens
Add MetricsStore in QML

Issue https://github.com/status-im/status-desktop/issues/15490

https://github.com/user-attachments/assets/8f60d102-0a11-4dbf-a7f2-319c0fb25ff5

https://github.com/user-attachments/assets/87e8c140-2bbe-43be-8ca7-25168d1d6679

### CODE REVIEW

File: ui/app/AppLayouts/Onboarding/views/KeysMainView.qml [hunk 1]
```diff
@@ -20,12 +23,13 @@ Item {
...
 
     property StartupStore startupStore
+    property MetricsStore metricsStore
 
     Component.onCompleted: {
         if (button1.visible) {
             button1.forceActiveFocus()
         }
-        enableCentricMetrics.checked = root.startupStore.isCentralizedMetricsEnabled()
+        metricsEnablePopup.visible = true
```
Comment (jrainville) [inline ui/app/AppLayouts/Onboarding/views/KeysMainView.qml:32]: I think we only want to show this popup on the first run, but it's worth checking with @xAlisher

File: ui/imports/shared/popups/MetricsEnablePopup.qml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+            AgreementSection {
+                title: qsTr("What we will receive")
+                body: qsTr(" •  IP address
+ •  Universally Unique Identifiers of device
+ •  Logs of actions within the app, including button presses and screen visits")
+            }
+
+            AgreementSection {
+                title: qsTr("What we not receive:")
```
Comment (jrainville) [inline ui/imports/shared/popups/MetricsEnablePopup.qml:78]: This sentence is not correct English :thinking: 
Is it supped to say `What we won't receive`?
Also, this one has a `:` but not the one above

**Review by jrainville — COMMENTED** [review]
Looks good, just two questions

File: ui/imports/shared/popups/MetricsEnablePopup.qml (hunk 2 above)
Comment (endulab) [reply ui/imports/shared/popups/MetricsEnablePopup.qml]: The text has changed.

File: ui/app/AppLayouts/Onboarding/views/KeysMainView.qml (hunk 1 above)
Comment (endulab) [reply ui/app/AppLayouts/Onboarding/views/KeysMainView.qml]: It is shown only during creating new account.

**Comment by anastasiyaig** [comment]
@endulab @jrainville we need to change tests for this PR to merge, we will work on this ASAP but not sure if we can do that immediately, perhaps it could be done tomorrow but @Valentina1133 is already checking on that

**Comment by endulab** [comment]
@jrainville @igor-sirotin @micieslak you can review again.

**Review by jrainville — APPROVED** [review]
Looks good. Nice work

**Review by igor-sirotin — COMMENTED** [review]
@endulab In your video the popup appears after `Get your keys` screen, but in Figma it's after `Welcome to Status`:
image

Were there some particular reasons for this? If not, I think it's better to have it after the first screen. And the code would be simpler.

Also, I guess if the user clicked `Not now`, we shouldn't show it once again if the user goes back onto previous page and then forward again. Not a very important, but might annoy users during onboarding.

**Comment by endulab** [comment]
Because in my next task I refactored the code and also adjusted UI according to a new design, I will close this pr in favor of https://github.com/status-im/status-desktop/pull/15685/files

**Pull request closed without merging**